extensions/Article/Entity/Article.xml

```diff
@@ -37,13 +37,6 @@
                description="COM_CONTENT_FIELD_ARTICLETEXT_DESC"
         />
 
-        <field name="author"
-               type="text"
-               label="COM_CONTENT_FIELD_CREATED_BY_ALIAS_LABEL"
-               description="COM_CONTENT_FIELD_CREATED_BY_ALIAS_DESC"
-               default=""
-        />
-
```

Comment:
Leave this field in the 'article' entity. It will be used as the 'author alias', when connected to 'users'.